src/main/java/it/gov/pagopa/pu/workflow/wf/receiptclassification/iuf/IufReceiptClassificationWFClient.java

```diff
@@ -0,0 +1,91 @@
+package it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf;
```

Comment:
just call it `classification`
```suggestion
package it.gov.pagopa.pu.workflow.wf.classification.iuf;
```